src/wallet/wallet.cpp

```diff
@@ -1408,13 +1407,25 @@ bool CWallet::IsHDEnabled()
     return !hdChain.masterKeyID.IsNull();
 }
 
+void CWallet::SetWalletFlag(uint64_t flag)
+{
+    walletFlags |= flag;
```

Comment:
IMO set flags seems most appropriate here. Flags are not getting *appended*. Either you set or unset IMO. For consistency, we should have a `UnsetWalletFlag()`.
But I agree its kind of confusing that we have a `SetWalletFlags` (plural) method that is capable or re-setting all 64 flags.